[2023-10-11 12:51:46,064] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 1_init_once_seed_data.load_seed_data_once manual__2023-10-11T12:51:45.260982+00:00 [queued]>
[2023-10-11 12:51:46,075] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 1_init_once_seed_data.load_seed_data_once manual__2023-10-11T12:51:45.260982+00:00 [queued]>
[2023-10-11 12:51:46,076] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-11 12:51:46,077] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-11 12:51:46,078] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-11 12:51:46,096] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): load_seed_data_once> on 2023-10-11 12:51:45.260982+00:00
[2023-10-11 12:51:46,166] {standard_task_runner.py:52} INFO - Started process 1442 to run task
[2023-10-11 12:51:46,175] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', '1_init_once_seed_data', 'load_seed_data_once', 'manual__2023-10-11T12:51:45.260982+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/init.py', '--cfg-path', '/tmp/tmpi6p1ikcv', '--error-file', '/tmp/tmp119udutn']
[2023-10-11 12:51:46,177] {standard_task_runner.py:80} INFO - Job 34: Subtask load_seed_data_once
[2023-10-11 12:51:46,244] {task_command.py:369} INFO - Running <TaskInstance: 1_init_once_seed_data.load_seed_data_once manual__2023-10-11T12:51:45.260982+00:00 [running]> on host c2e542998ec2
[2023-10-11 12:51:46,340] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=1_init_once_seed_data
AIRFLOW_CTX_TASK_ID=load_seed_data_once
AIRFLOW_CTX_EXECUTION_DATE=2023-10-11T12:51:45.260982+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-11T12:51:45.260982+00:00
[2023-10-11 12:51:46,342] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-10-11 12:51:46,343] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /dbt_snowflake && ls']
[2023-10-11 12:51:46,351] {subprocess.py:85} INFO - Output:
[2023-10-11 12:51:46,360] {subprocess.py:96} INFO - Command exited with return code 0
[2023-10-11 12:51:46,392] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=1_init_once_seed_data, task_id=load_seed_data_once, execution_date=20231011T125145, start_date=20231011T125146, end_date=20231011T125146
[2023-10-11 12:51:46,429] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-11 12:51:46,458] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
